release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1984,7 +1995,9 @@ oc create -f $PROJ-app.yaml
 +
 For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1812212[*BZ#1812212*].
 
-* Parsing undefined values results in a not a number (NaN) exception and the *Chart* tooltip shows a box with no values. To ensure that the results are synced and that undefined values are not parsed, specify a start date when fetching data so that the *Chart* toolip shows correct values. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1906304[*BZ#1906304*]. 
+* Parsing undefined values results in a not a number (NaN) exception and the *Chart* tooltip shows a box with no values. To ensure that the results are synced and that undefined values are not parsed, specify a start date when fetching data so that the *Chart* toolip shows correct values. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1906304[*BZ#1906304*].
```

Comment:
_specify a start date when fetching data_
From the BZ it appears this is an internal issue. This makes it sound like the problem is with the user entering improper information.

Reply:
Clarified and moved to bug fixes